src/services/DefectsService.ts

```diff
@@ -11,22 +13,46 @@ export class DefectsService {
   public getDefectList() {
     return this.defectsDAO
       .getAll()
-      .then((data: any) => {
+      .then(async (data: any) => {
```

Comment:
Not one to answer here, just making a note for the meeting later.

Is there any chance we can convert this method to using async/await rather than the .then syntax please. Appreciate it's just syntactic sugar but will make it more readable IMO.